Podfile

```diff
@@ -8,5 +8,6 @@ target 'Volume' do
   # Pods for Volume
   pod 'Apollo'
   pod 'SDWebImageSwiftUI'
+  pod 'AppDevAnalytics', :git => 'https://github.com/cuappdev/analytics-ios.git', :commit => '5d459c0475'
```

Comment:
Why a specific commit?

Reply:
I'm worried about needing to make breaking changes later in the semester. I have a few potential ideas to bump up the quality of the cocoapod, but need to experiment with a few apps first to see if its worth putting in the main pod.